.bazelignore

```diff
@@ -7,6 +7,7 @@ examples/linked_empty_node_modules/node_modules
 examples/linked_lib/node_modules
 examples/linked_pkg/node_modules
 examples/macro/node_modules/
+examples/nextjs/node_modules/
```

Comment:
we could get rid of this file and use `REPO.bazel` instead, it finally has wildcards for ignores

Reply:
We'll need to think about the bazelignore check and if we will remove that for v3 or support REPO.bazel instead etc.